### DISCUSSION TITLE
Fehler beim laden der Detailpage

### DISCUSSION THREAD

**Discussion by Marvin678** [neuefische/questions#249] [Web]
Moin Leute,

wenn ich meine detail Seite laden will, kommt diese Fehlermeldung.

![Bildschirmfoto 2022-10-12 um 13 50 04](https://user-images.githubusercontent.com/108797569/195335851-e1a48a4b-c14b-4164-8574-d82179fa2f38.png)

Ich dachte das es an meiner API liegen würde, da ich meine points schon aufgebraucht habe aufgrund der Anfragen die ich getätigt habe und habe dann meinen dritten Spoonacular Account erstellt, doch daran lag es auch nicht. Die Fehlermeldung bleibt die gleiche.

Bin dankbar für jede Hilfe die ich kriegen kann.
https://github.com/Marvin678/capstone-project/pull/6

**Comment by julialoeschel** [comment]
Hallo Marvin. 
Wie sieht denn deine Datenstruktur aus, die aus dem Fetch zurückkommt? gibt es da den key "results" ? 
lass dir das mal mit einem consoel.log(data) ausgeben und schau mal rein. 

Mareike hat mit verraten, das deine Struktur ein Array mit einem Objekt ist, in dem du den Key "steps" ansprechen musst. schau nochmal was ihr da gestern zusammen gemacht habt.
